crates/bitwarden-wasm-internal/src/client.rs

```diff
@@ -27,6 +27,56 @@ fn convert_level(level: LogLevel) -> Level {
     }
 }
 
+#[wasm_bindgen]
+pub struct BitwardenPure;
+
+#[wasm_bindgen]
+impl BitwardenPure {
+    pub fn version() -> String {
+        Self::setup_once();
+        env!("SDK_VERSION").to_owned()
+    }
+
+    pub fn echo(msg: String) -> String {
```

Comment:
```suggestion
```

Seems like left over debugging code?

Reply:
No I included those because they are present in the `BitwardenClient`, but more appropriately part of a pure function library.